component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -331,6 +334,10 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
         String contextToken = (String) context.getProperty(EmailOTPAuthenticatorConstants.OTP_TOKEN);
         long generatedTime = (long) context.getProperty(EmailOTPAuthenticatorConstants.OTP_GENERATED_TIME);
         boolean isExpired = isExpired(generatedTime, context);
+        /*Before proceeding further if the the email update is already failed in the last attempt it will be set to false*/
```

Comment:
incorrect comment format. add space after /* and before */. And end the comment with a fullstop

Reply:
Fix this issue